app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  validates :body, :user_id, :post_id, presence: true
```

Comment:
Dont use `:user_id` and `:post_id`..
Simply use, `:user` and `:post`.

Reply:
Thank you. It's been taken care of!